testing/src/main/java/io/stargate/it/http/GraphqlTest.java

```diff
@@ -827,6 +827,96 @@ public void insertProducts() {
     assertThat(product.getPrice()).hasValue(input.price());
     assertThat(product.getCreated()).hasValue(input.created());
     assertThat(product.getDescription()).hasValue(input.description());
+
+    // cleanup on the end
+    cleanupProduct(client, productId);
```

Comment:
move to an `@AfterEach` method?

Reply:
Yea, I was unaware you can get all products easily..  Done, resolving.

Reply:
We could refactor the test code to keep track of generated product IDs and then remove them all inside an `@AfterEach` method.

Reply:
no need, see here: https://github.com/stargate/stargate/blob/a39ec6014b9066ad091777c93bb5560a894a7949/testing/src/main/java/io/stargate/it/http/GraphqlTest.java#L181